packages/video_player/video_player_avfoundation/darwin/Classes/ios/FVPDisplayLink.m

```diff
@@ -0,0 +1,65 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "../FVPDisplayLink.h"
+
+#import <Foundation/Foundation.h>
+#import <QuartzCore/QuartzCore.h>
+
+/** A proxy object to act as a CADisplayLink target, to avoid retain loops. */
```

Comment:
can you explain more about the retain cycle situation and how this helps preventing it?

Reply:
Added more to the comment.